client/src/types/images.d.ts

```diff
@@ -0,0 +1,2 @@
+declare module '*.jpg';
+declare module '*.jpeg';
```

Comment:
Could you please make sure there's a single newline at the end of all JS/TS files? That way these red circles don't show up on git diffs as a different line when nothing changed.